tests/conftest.py

```
@@ -0,0 +1,12 @@
+from unittest.mock import patch
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def _disable_qualnames_in_namespaces(request):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_disable_qualnames_in_namespaces`

(ANN202)